students/kylee/users/utils.py

```diff
@@ -0,0 +1,27 @@
+import jwt
+
+from users.models import User
+from django.http  import JsonResponse
+
+from my_settings import (
+    MY_ALGORITMS,
+    MY_SECRET_KEY
+)
+
+def login_decorator(func) :
+    def wrapper(self, request, *args, **kwrags) :
+        try :
+            token = request.headers.get('Authorization', None)
+            payload = jwt.decode(token, MY_SECRET_KEY, MY_ALGORITMS)
+            login_user = User.objects.get(email=payload['email'])
```

Comment:
변수명은 간결하면서 명확하게 만들어주세요. 이미 login_decorator안에서 선언되어 있는 user이기 때문에 충분히 코드만 보고도 로그`user`가 login한 user임을 알 수 있습니다. `login_user`는 투머치!

```suggestion
            user = User.objects.get(email=payload['email'])
```